### ISSUE TITLE
[Story] Initiate collaboration with the Design team to review the UI before the private preview

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/sage-monorepo#1247]
### What projects is this story for?

OpenChallenges

### As a user, I want

As the product owner, I want the Design team to review our web UI so that the UX is optimized.

### Description

Contact Ljubo to initiate format collaboration to review the UI/UX of the OpenChallenges web app. The feedback from the Design team and the implementation of this feedback must be completed before then end of mid-April 2023. The private preview will be launched at the latest end of April.

### Acceptance criteria

- The Design team has been contacted.
- The scope of this collaboration has been described to the Design team.
- Ideally the Design team agree to collaborate.

Out of scope:
- Review of the UI/UX itself, which will be tracked in a separate ticket.

### Anything else?

- [OpenChallenges UI specification v3](https://www.figma.com/file/yAeIrg6Whc8Bqj2EnG4MWP/Challenge-Registry) (Figma)
- [OpenChallenges DACI model](https://docs.google.com/spreadsheets/d/1TRBHQzSxKWHek7EHdO9VJIcek27Y1q0dhmkhVVxByxI)

### Have you linked this story to a GitHub Project?

- [X] I have linked this story to a GitHub Project and set its metadata.

**Comment by tschaffter** [comment]
We don't have the bandwidth and time for a formal review process before the launch of the private preview. Instead, I'm hoping that we can identify and fix most of the UI issues/improvements by ourselves, with occasional feedback from others (e.g. Adam, Lawrence).
